scripts/demo.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+NO_COLOUR='\033[0m' # No Color
+
+EXAMPLES_DIR=${SCRIPT_DIR}/..
+
+resolveCommand () {
+  local targetCommand=${1}
+  local resolvedCommand
+  resolvedCommand=$(command -v "${targetCommand}")
+  if [[ -z ${resolvedCommand} ]]; then
+    echo -e "${RED}Unable to resolve path to ${targetCommand}${NO_COLOUR}" >&2
+    exit 127
+  else
+    echo "${resolvedCommand}"
+  fi
+}
+
+buildDataGenerator() {
+  echo -e "${GREEN}Building data generator${NO_COLOUR}"
+  ${MAVEN_COMMAND} -f "${EXAMPLES_DIR}/pom.xml" clean package
+  ${CONTAINER_ENGINE} build -f "${EXAMPLES_DIR}/data-generator/Dockerfile" -t flink-examples-data-generator:latest data-generator
+}
+
+installPrerequisites() {
+  echo -e "${BLUE}Installing ${OPERATORS} operators${NO_COLOUR}"
+
+  ${KUBE_COMMAND} apply -k "${EXAMPLES_DIR}/kubernetes-samples/supporting-infrastructure/overlays/${OPERATORS}_operators"
+
+  if [[ ${OPERATORS} == "Red_Hat" ]]; then
+    local AMQ_STREAMS_VERSION=""
```

Comment:
Should we use the new name instead of AMQ Streams?

Reply:
Fair point, we should use the proper name.